geomesa-blobstore/geomesa-blobstore-core/src/test/scala/org/locationtech/geomesa/blob/core/AccumuloBlobStoreTest.scala

```diff
@@ -38,10 +38,29 @@ class AccumuloBlobStoreTest extends Specification {
 
   val testfile1 = "testFile.txt"
   val testfile2 = "testFile2.txt"
+  val testfile3 = "testFile3.txt"
   var testFile1Id = ""
 
   "AccumuloBlobStore" should {
-    "be able able to store and retrieve a file" in {
+
+    "be able to store and delete a file" in pending("Deleting from accumulodatastore may not be working in blobstore") {
```

Comment:
deleting fails...debugger shoes that the modify feature writer is pulling out a feature with the feature id of ```1``` which probably indicates a user data problem or something like that...